content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -28,6 +26,15 @@ containers, the timezone set for the kube-controller-manager container determine
 that the cron job controller uses.
 {{< /caution >}}
 
+{{< caution >}}
+CronJob does not officially support setting timezone as explained above.
+
+Setting variables such as `CRON_TZ` is not officially supported.
+`CRON_TZ` is an implementation detail of the internal library being used for
+parsing and calculating the next Job creation time. Any usage of it is not
+recommended in a production cluster.
+{{< /caution >}}
+
```

Comment:
We might want to update the section headed “Cron schedule syntax” too.

Reply:
Good catch, updated.